consensus/poet/core/sawtooth_poet/poet_consensus/poet_block_verifier.py

```diff
@@ -183,6 +184,65 @@ def verify_block(self, block_wrapper):
                             validator_state.key_block_claim_count,
                             poet_config_view.key_block_claim_limit))
 
+            # While having a block claim delay is nice, it turns out that in
```

Comment:
Can do.  My plan was that because these three PRs are intertwined, that in the third PR I was going to refactor the PoET policy tests to be in one place and the PoetBlockVerifier and PoetBlockPublisher would call them.